### ISSUE TITLE
make buildglyphimages.py more robust

### ISSUE THREAD

**Issue by uliska** [uliska/lilyglyphs#34]
The script isn't robust in several respects. But I wanted to make it usable at least for my personal use first, and make it robust later.

Milestone 1.0 is only the last resort. It should definitely be fixed earlier.
- the parser of the input file just assumes that this is quite correct, not much error handling is implemented. It should be either made smarter (in order to deal with sloppy input) or at least check for well-formedness and exit safely and informatively) if it encocunters unusable input.
- The script makes several assumptions on the existing directory structure. If they turn out to be wrong I don't know what will happen - maybe the script just stops and leaves a half-finished state. (This probably won't do much harm, though). We should insert a thorough check before starting to write anything out onto disk, and also implement error handling on the way
- ATM the script overwrites any present files (if an entry isn't 'protected'). The presence of a -f / --force option isn't used so far. Probably there should be some way to selectively overwrite existing files. OTOH I'm not sure yet if we can say that the created files should _not_ be updated manually at all.
